osm2gtfs/core/osm_connector.py

```diff
@@ -592,6 +592,7 @@ def _query_routes(self):
 
             /* Return tags for elements and roles for relation members. */
             );out body;""" % (self.tags, self.bbox)
+        print(query_str)
```

Comment:
You forgot to remove the print here.

Reply:
I put this on purpose, in order to have the query printed as part of the general output.

Reply:
Moved over to PR #133.